operator/src/main/java/oracle/kubernetes/operator/ProcessingConstants.java

```diff
@@ -34,4 +34,6 @@
   public static final String DOMAIN_INTROSPECT_VERSION = "weblogic.domainIntrospectVersion";
   public static final String DOMAIN_INPUTS_HASH = "weblogic.domainInputsHash";
   public static final String REMAINING_SERVERS_HEALTH_TO_READ = "serverHealthRead";
+
+  public static final String KEY = "encodedCredentials";
```

Comment:
You need a better constant name.  Why not ENCODED_CREDENTIALS?

Reply:
> You need a better constant name. Why not ENCODED_CREDENTIALS?

Yes, I'll change it to ENCODED_CREDENTIALS. The constant name was previously used to store httpClient instance in packet and I was lazy to use same name for encoded credentials

Reply:
Fixed in commit ba303f0.